homeassistant/components/freebox/config_flow.py

```diff
@@ -20,45 +21,35 @@ class FreeboxFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    def __init__(self) -> None:
-        """Initialize Freebox config flow."""
-        self._host: str
-        self._port = None
+    _data: dict[str, Any] = {}
```

Comment:
Please avoid initializing mutable objects as class attributes. It's error prone. Use an init method for those to make them instance attibutes.

Reply:
It's Python basic rules. If there's a class attribute that is mutable all instances of the class will see changes to the mutable object. That can lead to hard-to-find bugs.

Reply:
Can you kind of compare it to a singleton but as attribute?